src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1199,7 +1253,9 @@ public void Dispose()
             CustomNodeManager.MessageLogged -= LogMessage;
             CustomNodeManager.Dispose();
             MigrationManager.MessageLogged -= LogMessage;
-
+            FeatureFlags.MessageLogged -= LogMessageWrapper;
+            DynamoUtilities.DynamoFeatureFlagsManager.FlagsRetrieved -= CheckFeatureFlagTest;
+            FeatureFlags.Dispose();
```

Comment:
Isn't it going to be disposed at the end of the `using` anyway?

Reply:
yes, this is a leftover from the other implementation, will get rid of it.